autosynth/synth.py

```diff
@@ -144,6 +144,7 @@ def __init__(
         temp_dir: str,
         metadata_path: str,
         synth_path: str,
+        log_path: pathlib.Path = None,
```

Comment:
Because it's a path to a directory, let's call it `log_dir_path`.

Reply:
Doine